src/components/Expenses/Expense.js

```diff
@@ -0,0 +1,53 @@
+import ExpenseItem from "../Expenses/ExpenseItem";
+import NewExpense from "../NewExpense/NewExpense";
+
+const Expense = () => {
+  const expenses = [
+    {
+      id: "e1",
+      title: "Toilet Paper",
+      amount: 94.12,
+      date: new Date(2020, 7, 14),
+    },
+    { id: "e2", title: "New TV", amount: 799.49, date: new Date(2021, 2, 12) },
```

Comment:
can you format it like others?? 
For eg: 
 { 
       id : "e2",
       title: "New TV",
        .........
}

Reply:
Done! Thanking You !